sanowret-crystal.lic

```diff
@@ -0,0 +1,73 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
+
+	author: NBSL
+	game: dr
+  version: 0.1 (2017-06-12)
+
+  changelog:
+    0.1 (2017-06-12)
+     * Initial Release
+=end
+
+custom_require.call(%w[common drinfomon])
+
+
+class SanowretCrystal
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'run', regex: /run/i, description: 'Single run of the script', optional: true }
+      ]
+    ]
+    @invalid_room = nil
+    args = parse_args(arg_definitions)
+    check_crystal if args.run
+    passive if args.run.nil?
```

Comment:
You could also `passive unless args.run`

Though I wonder if this could be combined: always go into the loop, exit after the first iteration if the arg was used